static/js/components/RepeatableContentListing.tsx

```diff
@@ -113,9 +114,9 @@ export default function RepeatableContentListing(props: {
       <div>
         <Card>
           <div className="d-flex flex-direction-row align-items-center justify-content-between pb-3">
-            <h3>{configItem.label}</h3>
+            <h3>{labelSingular}</h3>
```

Comment:
Since this will be the heading for a list of items (pages, resources, etc), should the plural still be used here?

Reply:
Looks like the issue shows the plural here - https://user-images.githubusercontent.com/125561/122150900-95150e00-ce2c-11eb-9bd2-3afef3513cf2.png

Reply:
Thanks, just fixed it